source/AI.cpp

```diff
@@ -1509,6 +1509,28 @@ bool AI::FollowOrders(Ship &ship, Command &command) const
 
 
 
+bool AI::ShouldStay(Ship &ship) const
```

Comment:
Can the `ship` reference passed in be `const`?
```suggestion
bool AI::ShouldStay(const Ship &ship) const
```

Reply:
Yes, and yes. It is now in the local namespace with a `const Ship &` argument.